src/axom/spin/tests/CMakeLists.txt

```diff
@@ -33,38 +33,46 @@ blt_list_append( TO spin_depends ELEMENTS cuda IF ${ENABLE_CUDA} )
 
 foreach ( test ${spin_tests} )
 
-   get_filename_component( test_name ${test} NAME_WE )
+    get_filename_component( test_name ${test} NAME_WE )
 
-   blt_add_executable(
-        NAME       ${test_name}_test
-        SOURCES    ${test}
-        OUTPUT_DIR ${TEST_OUTPUT_DIRECTORY}
-        DEPENDS_ON ${spin_depends}
-        FOLDER     axom/spin/tests
-       )
+    blt_add_executable(
+      NAME       ${test_name}_test
+      SOURCES    ${test}
+      OUTPUT_DIR ${TEST_OUTPUT_DIRECTORY}
+      DEPENDS_ON ${spin_depends}
+      FOLDER     axom/spin/tests
+      )
 
-   blt_add_test(
-        NAME    ${test_name}
-        COMMAND ${test_name}_test
-       )
+    axom_add_test(
+      NAME    ${test_name}
+      COMMAND ${test_name}_test
+      )
 
 endforeach()
 
 if ( RAJA_FOUND AND UMPIRE_FOUND )
 
-   blt_add_executable(
+    blt_add_executable(
       NAME       spin_bvh_test
       SOURCES    spin_bvh.cpp
       OUTPUT_DIR ${TEST_OUTPUT_DIRECTORY}
       DEPENDS_ON mint ${spin_depends}
       FOLDER     axom/spin/tests
-     )
+      )
 
-   blt_add_test(
-      NAME    spin_bvh
-      COMMAND spin_bvh_test
+    axom_add_test(
+      NAME            spin_bvh
+      COMMAND         spin_bvh_test
       NUM_OMP_THREADS 4
-     )
+      )
+
+    # fix for rzmanta of following runtime error:
+    #   CUDAassert: peer access is not supported between these two devices /unimportant/path/raja-0.12.1/include/RAJA/policy/cuda/sort.hpp 341
+    #   unknown file: Failure
+    #   C++ exception with description "CUDAassert" thrown in the test body.
+    set_property(TEST spin_bvh
```

Comment:
👍

Reply:
Would it be worthwhile to fold `ENVIRONMENT` into `blt_add_test` or `axom_add_test`?

(Clearly this in not required, but might be a nice convenience?)

Reply:
https://github.com/LLNL/blt/issues/443